File: kitten-housing-care/src/main/java/org/jesperancinha/housing/converter/OwnerConverter.java
package org.jesperancinha.housing.converter;

import org.jesperancinha.housing.data.OwnerDto;
import org.jesperancinha.housing.model.Owner;

public class OwnerConverter {
    public static OwnerDto toDto(Owner owner) {
        return OwnerDto.builder()
                .name(owner.getName())
                .addres(owner.getAddres())
                .build();
    }
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/converter/OwnerConverter.kt
package org.jesperancinha.housing.converter

import org.jesperancinha.housing.data.OwnerDto
import org.jesperancinha.housing.model.Owner

object OwnerConverter {
    fun toDto(owner: Owner?): OwnerDto {
        return OwnerDto.builder().name(owner.getName()).addres(owner.getAddress()).build()
    }
}